sui-execution/v0/sui-move-natives/src/object_runtime/object_store.rs

```diff
@@ -118,10 +118,13 @@ impl<'a> Inner<'a> {
                             ))
                         }
                     }
-                    Owner::AddressOwner(_) | Owner::Immutable | Owner::Shared { .. } => {
+                    Owner::AddressOwner(_)
+                    | Owner::Immutable
+                    | Owner::Shared { .. }
+                    | Owner::ConsensusV2 { .. } => {
```

Comment:
same, can be a separate match arm that panic

and same for pretty much everything below it